requirements/constraints.txt

```diff
@@ -13,3 +13,6 @@ more-itertools<6.0.0
 
 # Newer versions of pylint-plugin-utils drop Python 2.7 support
 pylint-plugin-utils<0.4
+
+# requests only supports 1.24 and selenium tries to pull in latest
```

Comment:
Maybe link to the `requests` issue for resolving this? https://github.com/kennethreitz/requests/issues/5067